src/Build/Construction/ProjectImportElement.cs

```diff
@@ -75,13 +76,41 @@ public string Sdk
             set
             {
                 ErrorUtilities.VerifyThrowArgumentLength(value, XMakeAttributes.sdk);
-
+                if (!CheckUpdatedSdk()) return;
                 ProjectXmlUtilities.SetOrRemoveAttribute(XmlElement, XMakeAttributes.sdk, value);
                 MarkDirty("Set Import Sdk {0}", value);
             }
         }
 
         /// <summary>
+        /// Gets or sets the version associated with this SDK import
+        /// </summary>
+        public string Version
```

Comment:
Add a new test to [construction editing tests](https://github.com/Microsoft/msbuild/blob/master/src/Build.OM.UnitTests/Construction/ConstructionEditing_Tests.cs) to assert null values, non-null values, etc. For example, what happens if you add an SDK import element inside an import group group together with non SDK import elements.